cypress-tests/cypress/e2e/updateProjects.cy.ts

```diff
@@ -13,9 +13,9 @@ const projects = {
 
 // ? to simplify debugging, you can change `shouldFork` to false to use the projects directly instead of forking.
 // projects.shouldFork = false;
-// projects.namespace = youtnamesoace;
-// projects.v7 = another-project-v7;
-// projects.v8 = another-project-v8;
+// projects.namespace = "yourNamespace";
```

Comment:
noooo why are we committing commented code?!?! so many sad pandas!!!

Reply:
ha I was about to write the same (well maybe minus the pandas) - can't we just force the tests to pass (with an appropriate comment) and leave the rest as-is, especially if we plan to use it again?

Reply:
Renku core currently supports only v9, there is no project we (nor the users) can update automatically atm :smile: 
It's just commented out, it will come back as soon as renku-core v2 is merged (it should support v9 and v10)

Reply:
yes, this I understand, but can't you just short-circuit the test instead of the giant comment block?

Reply:
Wdym by "short-circuit the test"?
Btw this will last until renku-core v2 is merged, so I guess it's a matter of days

Reply:
I just (naively) mean to return at the top of the function...

Reply:
Mmm that seems worse to me tbh :D It's easier to forget it's unused.

Reply:
guys this is **GIT**: just delete the code (in its own commit) and revert the commit once it's ready to go back in again!

Reply:
I've removed the unusable test (I didn't consider leaving some commented code temporarily would be a big deal)
I left the comments on the top for testing on the fly with a custom project. I believe that's useful, especially for people who need to debug something on the fly.

Reply:
thank you!